sky/provision/kubernetes/network_utils.py

```diff
@@ -75,13 +75,16 @@ def fill_loadbalancer_template(namespace: str, service_name: str,
 
     with open(template_path, 'r', encoding='utf-8') as fin:
         template = fin.read()
+    annotations = skypilot_config.get_nested(
+        ('kubernetes', 'custom_metadata', 'annotations'), {})
```

Comment:
We probably want to do this for `labels` as well. Could we add that in this PR? Or put a TODO and we can defer it to another PR.

Reply:
done!